### DISCUSSION TITLE
lush is awesome!

### DISCUSSION THREAD

**Discussion by karimknaebel** [rktjmp/lush.nvim#45] [General]
Just wanted to thank you for making this project. I honestly can't believe how good both the plugin itself, and the documentation/tutorial is. The live preview and DSL are a joy to work with. Thank you!

**Comment by rktjmp** [comment]
Glad you are having fun with it. I never thought anyone else but me would really use it. :+1:

Hopefully when I get time to work on #36 Lush will be able to fit better with complex themes and be even better.
